test/run/test_stages.py

```diff
@@ -658,3 +658,36 @@ def test_erofs(self):
                 check=True,
             )
             assert "testfile" in r.stdout
+
+    def test_bootc_install_to_fs(self):
+        datadir = self.locate_test_data()
+        testdir = os.path.join(datadir, "stages", "bootc.install-to-fs")
+
+        with self.osbuild as osb, tempfile.TemporaryDirectory(dir="/var/tmp") as outdir:
+            osb.compile_file(os.path.join(testdir, "manifest.json"), exports=["image"], output_dir=outdir)
+
+            image = os.path.join(outdir, "image", "disk.img")
+            assert os.path.isfile(image)
+
+            # check that we have a basic boot/ostree dir layout on /
+            env = os.environ.copy()
+            env["LIBGUESTFS_BACKEND"] = "direct"
+            # check xattrs
```

Comment:
Thank you, this is excellent feedback. I had to remove the test (for now) because of hitting issues with how our testing is done but I will keep your comments in mind. I mostly did it this way to avoid that by booting the image I alter the state by booting but agree that actually booting it is probably a (much) better test as it tests more (and more of the right things).

Once this is used from bib we will get the bib based integration testing which does actual boot tests.